eventsources/sources/webhook/start.go

```diff
@@ -155,3 +162,56 @@ func (el *EventListener) StartListening(ctx context.Context, dispatch func([]byt
 		route: route,
 	}, controller, dispatch)
 }
+
+func GetBody(writer *http.ResponseWriter, request *http.Request, route *webhook.Route, logger *zap.SugaredLogger) (*json.RawMessage, error) {
+	request.Body = http.MaxBytesReader(*writer, request.Body, 65536)
```

Comment:
Shall this move the `default` content-type section below? Does it make any difference for the rest?

Reply:
Thank you for the comment. Yes, we can move to inside the `case` section. However, if we want to add more content-type like `text/csv`, this method will be used. So I want to make this function separated.

But I don't have a strong motivation, so I can move to inside.

Reply:
Let's move inside for now, that could be changed if we support more use cases in the future.

Reply:
Updated!

Reply:
hmmm... there might be some misunderstanding :) I meant to move the line below closer to the place that `request.Body` gets read.

```
request.Body = http.MaxBytesReader(*writer, request.Body, 65536)
```

Reply:
Ah, sorry for my mistake. I revert the function move change and move the `MaxBytesReader` lines to inside default case.